bindata/bootkube/manifests/etcd-service-endpoints.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: etcd
+  namespace: kube-system
+subsets:
+- addresses:
+  - ip: {{ .MasterIPAddress }}
```

Comment:
Does this have to be an IP address?  Can't we use DNS or something less brittle?

Reply:
For example, [we're already pushing etcd endpoints][1] which the templates can consume.  Is there a reason you can't use them?  (I don't actually understand what this PR is trying to accomplish, so take my feedback with large grains of salt ;).

[1]: https://github.com/openshift/installer/blob/669ce4419d2ce1d79a8aacbe8a29c39dfeff7f5c/pkg/asset/ignition/bootstrap/bootstrap.go#L143-L146

Reply:
@wking it means you use the DNS provided by the cloud provider instead of in-cluster DNS, right?

Reply:
Correct, it would not be using in-cluster DNS. And also agree with @wking  - we should use the same list of etcd endpoints that we already have available